src/Operation.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Aidantwoods\SecureHeaders;
+
+interface Operation
+{
+    /**
+     * Transform the given set of headers
+     *
+     * @param HeaderBag $headers
+     * @return void
+     */
+    public function modify(HeaderBag $headers);
```

Comment:
I know it's not necessary to work, my point is it's more clear for the reader ;p

(The fact that the actual behaviour can only be determined by tests or reading [stack overflow](http://stackoverflow.com/a/9696799) answers makes PHPs object 'value' interpretation pretty uncommon knowledge)